pkg/pci/pci_linux.go

```diff
@@ -233,7 +238,21 @@ func (info *Info) GetDevice(address string) *Device {
 	if modaliasInfo == nil {
 		return nil
 	}
+	return info.getDeviceFromModaliasInfo(address, modaliasInfo)
+}
+
+// ParseDevice returns a pointer to a Device given its describing data.
+// The PCI device obtained this way may not exist in the system;
+// use GetDevice to get a *Device which is found in the system
+func (info *Info) ParseDevice(address, modalias string) *Device {
```

Comment:
technically this is introducing a public exported interface, and I realize you did this in order to unit-test it in the pci_test.go file which imports the `pkg/pci` package in order to test the exported interfaces. So, I'm OK with this.